### PR TITLE
Add Generics and Nullability Qualifiers to KADOperation

### PR DESCRIPTION
**Pull request by mazz**

### CODE REVIEW

File: Operations/Operations/KADOperation.m [hunk 1]
```diff
@@ -172,13 +172,13 @@ -(void)cancel
 {
     [self cancelWithError:nil];
 }
--(void)cancelWithError:(NSError *)error
+-(void)cancelWithError:(NSError __nullable*)error
 {
 
     [self.internalErrors addObject:error];
```
Comment (kas-kad) [inline Operations/Operations/KADOperation.m:178]: oh nice one. Inserting nullable into an array. Will fix this.

**Comment by kas-kad** [comment]
Oh, it would be great for you to update your XCode to 6.4 http://monosnap.com/image/P52q5Pz0pWYulqzM29FViZNhZ5J3sS.png

**Comment by purrrminator** [comment]
I've reset the master to previous commit to cancel your pull request.

**Comment by kas-kad** [comment]
Wait. Do you use XCode 7?

**Comment by kas-kad** [comment]
This is from my XCode 7 http://monosnap.com/image/gUJJpuEJMDkolP0ZFtFFMibPIEBrSJ.png

**Pull request merged**